.gitignore

```diff
@@ -1,6 +1,15 @@
 # Byte-compiled / optimized / DLL files
 __pycache__/
 
+# OS
+.DS_Store
+
+# Editors
+.idea
+*.sublime-project
+*.sublime-workspace
+*.sublime-workspace
+
```

Comment:
you can put this in your global .gitignore, e.g.: `.gitignore_global`:

```bash
git config --global core.excludesfile ~/.gitignore_global
```

see https://help.github.com/articles/ignoring-files/#create-a-global-gitignore for instance